core/quorum/block_voting.sol

```diff
@@ -13,27 +13,27 @@ contract BlockVoting {
     // Raised when a new address is allowed to vote.
```

Comment:
In the vote event, please change  the name of the param from "blockNumber" to "height" to match the name of the expected value.